cmd/meroxa/root/builds/builds.go

```diff
@@ -46,13 +43,9 @@ func (*Builds) Hidden() bool {
 	return true
 }
 
-func (*Builds) FeatureFlag() (string, error) {
-	return "turbine", fmt.Errorf("no access to the Meroxa Data Application feature")
-}
-
 func (*Builds) Docs() builder.Docs {
 	return builder.Docs{
-		Short: "Manage Process Builds on Meroxa",
+		Short: "Operate with Process Builds on Meroxa",
```

Comment:
I think this update is less clear than the original.

Reply:
@janelletavares ok. I'll change it back since I don't feel strongly opinionated either way. My thinking was, there's nothing to manage on a build (I cannot create, update or delete), but only describe or logs. Maybe `inspect` would make more sense. I'll change it back to `Manage` and if you agree with `Inspect` I'll change it in another commit / pr.

Reply:
Inspect makes sense.  View.  Describe.  Any of those are good with me.